docs/sections/user_guide/cli/drivers/filter_topo/help.out

```diff
@@ -12,6 +12,8 @@ Optional arguments:
 
 Positional arguments:
   TASK
+    filtered_output_file
+      The filtered output file staged from raw input
```

Comment:
This ended up being required in the rundir for filter_topo. It's a copy of the raw orog output since it modifies the file in place.